datanode/gateway/graphql/schema.graphql

```diff
@@ -1906,7 +1902,7 @@ type Party {
   accounts(
     "Market ID - specify what market accounts for the party to return"
     marketId: ID
-    "Asset (ETH, DAI etc)"
+    "Asset symbol (ETH, DAI etc)"
```

Comment:
Is the schema fibbing here because in this query 'asset' is an object not a string...
{  parties {  accounts{    asset {      id    }   }  } }

Reply:
Oh I am being dumb - this is the query arg not the return type!